tests-scan

```diff
@@ -287,6 +291,21 @@ def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp
         else:
             logging.error("Can't find pull request %s", pull_number)
             return 1
+    elif sha:
+        pulls.append({
+            "title": f"{sha}",
+            "number": 0,
+            "head": {
+                "sha": sha,
+                "user": {
+                    "login": "cockpit-project"
+                }
+            },
+            "base": {
+                "ref": testmap.get_default_branch(repo)
+            },
+            "labels": [{"name": "main"}],
```

Comment:
Ah, does this also need to change to get_default_branch()? If that is a github label, I'm not sure why it would even be called "main", instead of the "nightly" one you defined in tests-invoke?